controllers/workspace/devworkspace_controller.go

```diff
@@ -156,17 +156,6 @@ func (r *DevWorkspaceReconciler) Reconcile(req ctrl.Request) (reconcileResult ct
 		return reconcile.Result{Requeue: true}, err
 	}
 
-	restrictedAccess := clusterWorkspace.Annotations[config.WorkspaceRestrictedAccessAnnotation]
```

Comment:
would it be a good idea to check if webhooks really exists on the cluster here? Or when they absent workspace can't be reconciled due `validateCreatorLabel` check? 
I think the second.

Reply:
I think it's probably safer to do some check that webhooks are enabled (i.e. actually checking that webhooks exist on the cluster), but I would separate that out into another issue.

Reply:
Created https://github.com/devfile/devworkspace-operator/issues/305